doc/services.md

```diff
@@ -0,0 +1,42 @@
+# Build WG Services
+
+This document is intended to outline all services managed by the Build WG,
+and how to configure and operate them.
+
+### Ansible configuration
+
+Ansible is the tool used to managed all machines that the Build WG is responsible for.
+
+Secrets for Ansible scripts live within the `build` directory of the [`nodejs-private/secrets`](https://github.com/nodejs-private/secrets/tree/master/build) repository. Access to credentials is granted upon joining the build team
```

Comment:
Don't people need to open a PR adding their public key there to actually get the access?

Reply:
New members have to send their keys to people that already have access who then run [`dotgpg add`](https://github.com/ConradIrwin/dotgpg#dotgpg-add) to add them